snare/startup.py

```diff
@@ -0,0 +1,93 @@
+import asyncio
```

Comment:
there is no need to move these code into separate file, you can do it in the bin file